src/libraries/System.Formats.Nrbf/src/System/Formats/Nrbf/SerializationRecordType.cs

```diff
@@ -6,12 +6,6 @@ namespace System.Formats.Nrbf;
 /// <summary>
 /// Record type.
 /// </summary>
-/// <remarks>
-///  <para>
-///   The enumeration does not contain all values supported by the <see href="https://learn.microsoft.com/openspecs/windows_protocols/ms-nrbf/954a0657-b901-4813-9398-4ec732fe8b32">
-///   [MS-NRBF] 2.1.2.1</see>, but only those supported by the <see cref="NrbfDecoder"/>.
```

Comment:
Is this untrue now?

Reply:
Fair enough.  I noticed at least one member saying it wasn't supported, but it also looked like some numbers got skipped and I didn't crack open the spec to compare.